annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -90,7 +90,16 @@
             "splice_acceptor_variant", "splice_donor_variant", "splice_region_variant"));
 
     private Map<String, String> variantMap = new HashMap<String, String>();
-    private List<String> hgvspNullClassifications = new ArrayList<>();
+    private static List<String> hgvspNullClassifications = new ArrayList<>();
```

Comment:
This is a nice pattern for initializing static members:

private static List<String> hgvspNullClassifications = initNullClassifications();
private static List<String> initNullClassifications()
{
...
}